### PR TITLE
Setup info for umbrella apps.

### PR DESCRIPTION
**Pull request by wzph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +119,62 @@ Then in your `test_helper.exs` you can provide some configuration to Wallaby. At
 Application.put_env(:wallaby, :base_url, YourApplication.Endpoint.url)
 ```
 
+#### Umbrella Apps
+
+If you're testing an umbrella application containing a Phoenix application for
+the web interface (`MyWebApp`) and a separate persistence application
+(`MyPersistenceApp`) using Ecto 2.0 with a database that supports
```
Comment (PragTob) [inline README.md]: Maybe just say Ecto 2 as it also holds true for 2.1+ I suspect :)

File: README.md [hunk 2]
```diff
@@ -118,6 +119,62 @@ Then in your `test_helper.exs` you can provide some configuration to Wallaby. At
...
+sandbox mode, then you can use the same setup as above, with a few tweaks.
+
+```elixir
+# my_web_app/lib/endpoint.ex
+
+defmodule YourApp.Endpoint do
```
Comment (PragTob) [inline README.md]: SHouldn't this be `MyWebApp` to keep the terms as above or change them above?

File: README.md [hunk 3]
```diff
@@ -118,6 +119,62 @@ Then in your `test_helper.exs` you can provide some configuration to Wallaby. At
...
+  [
+    {:wallaby, "~> 0.17.0", only: :test},
+    {:phoenix_ecto, "~> 3.0", only: :test}
+  ]
+end
+```
+
+
+
+
```
Comment (PragTob) [inline README.md]: I don't think we need that many blank lines here :)

**Comment by PragTob** [comment]
Also, thank you very much!

nala_yum_yum

**Comment by wzph** [comment]
You got it!

**Pull request merged**